src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -27,6 +30,18 @@ public void actionPerformed(ActionEvent e) {
 				 FiguresJoe.figureScript2(driverManager.getCurrentDriver());
 				 break;
 			}
+			case 3:{
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.